sdk/tables/aztable/table_service_client.go

```diff
@@ -68,13 +68,24 @@ func (t *TableServiceClient) Delete(ctx context.Context, name string) (TableDele
 // pager := client.Query(nil)
 // for pager.NextPage(ctx) {
 //     resp = pager.PageResponse()
-//     fmt.sprintf("The page contains %i results", len(resp.TableQueryResponse.Value))
+//     fmt.Fprintf("The page contains %i results", len(resp.TableQueryResponse.Value))
```

Comment:
Thorough!

Reply:
Although....shouldn't it just be fmt.Printf?

Reply:
Butting in to your thread 😝 It should be `fmt.Sprintf(..)`

Reply:
I'm going to go with `fmt.Printf` because the goal is just to print to stdout the results.

Reply:
@catalinaperalta  - is this because of the kate-a-line-UH thing? I have already apologized for that debacle.

Reply:
That may have been a contributing factor...